### PR TITLE
Copy a list of objects from GCS to S3

### PR DESCRIPTION
**Pull request by alepuccetti**
This PR contains some not merged code 🤦  but more important is the new operator to copy a list of file from GCS to S3 this will be used to copy over the feed exported files.

Some things worth mentioning:
- `GoogleCloudStorageToS3CopyObjectListOperator` won't fail if a source file does not exist but only print a message.
This was necessary to handle the case where a country does not contain any data in a single day and we know that only after the files are generated. This operator was implemented in this way to handle the data generated by the [bigquery-query-results-gcs-exporter](https://github.com/huq-industries/bigquery-query-results-gcs-exporter).
- `GoogleCloudStorageToS3CopyOperator` will fail if the source file does not exist. This is mainly because this operator copy only a single file and we use this operator that we know it should be there.

~This is a draft until we test the code on the new composer environment after re-run is finished.~ The code now is tested and running in the production Mozart environment. ✅ 

Also, a lot of code has been removed because we do not use it anymore and it was an hack to overcome the issue of creating thousand of short-living tasks.

### CODE REVIEW

File: huq/gcs.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def __init__(self,
                  delegate_to=None,
                  *args,
                  **kwargs):
-        super(GoogleCloudStorageComposePrefixOperator, self).__init__(*args, **kwargs)
+        super(self).__init__(*args, **kwargs)
```
Comment (turbomerl) [inline huq/gcs.py]: in fact I think it can just be super().__init__()

File: huq/gcs.py [hunk 2]
```diff
@@ -175,28 +175,26 @@ def execute(self, context):
...
                  dest_verify=None,
                  *args,
                  **kwargs):
 
-        super(GoogleCloudStorageToS3CopyChainOperator, self).__init__(*args, **kwargs)
-        self.gcs_source_buckets = gcs_source_buckets
-        self.gcs_source_objects = gcs_source_objects
-        self.s3_destination_buckets = s3_destination_buckets
-        self.s3_destination_uris = s3_destination_uris
+        super().__init__(*args, **kwargs)
```
Comment (turbomerl) [inline huq/gcs.py:200]: should always run super() this way

Comment (turbomerl) [inline huq/gcs.py:200]: like this

File: dbt_cloud/operators/dbt_cloud.py [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+            raise AirflowException(f'Error deleting job {self.job_id}: {e}')
+
+        return job.get('id')
+
+
+class DbtCloudTriggerRunOperator(BaseOperator):
+    """
+    Trigger a run for a dbt cloud job.
+
+    We use job_id as either job_id or a task)id that return a job_id.
```
Comment (turbomerl) [inline dbt_cloud/operators/dbt_cloud.py:198]: task_id

File: dbt_cloud/operators/dbt_cloud.py [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+            raise AirflowException(f'Error updating job {self.job_id}: {e}')
+
+        return job.get('id')
+
+
+class DbtCloudDeleteJobOperator(BaseOperator):
+    """
+    Delete dbt cloud job
+
+    We use job_id as either job_id or a task)id that return a job_id.
```
Comment (turbomerl) [inline dbt_cloud/operators/dbt_cloud.py:160]: task_id

**Review by turbomerl — COMMENTED** [review]
couple of super tiny comments - looks good

File: huq/gcs.py (hunk 1 above)
Comment (alepuccetti) [reply huq/gcs.py]: Probably a bit of code slipped trough the cracks and got committed ba471e3440c0cd421789b7af6993d2dd80a9e62e.

**Review by turbomerl — APPROVED** [review]
great

**Pull request merged**